frontend/TODO.md

```
@@ -7,14 +7,14 @@
         - [ ] Verify compile moment thru FEJSON
     - [ ] User action reporting
         - [ ] New post creation
-        - [ ] Log out
+        - [x] Log out
```

Markdownlint (0.37.0)

Expected: 4; Actual: 8
Unordered list indentation

(MD007, ul-indent)